infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -308,6 +330,25 @@ public void multiRead(CorfuPayloadMsg<MultipleReadRequest> msg, ChannelHandlerCo
         }
     }
 
+    @ServerHandler(type = CorfuMsgType.INSPECT_ADDRESSES_REQUEST)
+    public void inspectAddresses(CorfuPayloadMsg<InspectAddressesRequest> msg,
+                                 ChannelHandlerContext ctx, IServerRouter r) {
+        log.trace("inspectAddresses: {}", msg.getPayload().getAddresses());
+        InspectAddressesResponse inspectResponse = new InspectAddressesResponse();
+        try {
+            for (long address : msg.getPayload().getAddresses()) {
```

Comment:
code reformat please

Reply:
There is nothing to format here, the entire method is already well formatted. Even IDE tells it's already formatted.